src/app/containers/Bulletin/index.test.jsx

```diff
@@ -96,9 +96,10 @@ describe('Bulletin Container', () => {
       const { container } = render(BulletinWithContext(liveTvBulletinItem));
       const span = container.getElementsByTagName('span')[2];
 
-      expect(span.getAttribute('aria-hidden')).toBeDefined();
-      expect(span.getAttribute('aria-hidden')).toEqual('true');
-      expect(span.textContent).toEqual('LIVE');
+      expect(span.textContent).toEqual('NA EME UGBU A');
+      if (span.textContent === 'LIVE') {
```

Comment:
I don't think this will test anything? We can add another test to do this though

Reply:
This actually won't happen at the moment in Bulletin - we have a seperate ticket to amend how the visually hidden text works in here.